app/controllers/work_packages/date_picker_controller.rb

```diff
@@ -157,7 +157,29 @@ def datepicker_modal_component
                                                          focused_field:,
                                                          triggering_field: params[:triggering_field],
                                                          touched_field_map:,
-                                                         date_mode:)
+                                                         date_mode:,
+                                                         live_region_message: live_region_message)
```

Comment:
Nitpick. You can use shorthand Hash syntax here:

```suggestion
                                                         live_region_message:)
```